ecf/scripts/plots/global_ens/jevs_global_ens_atmos_gefs_grid2obs_past90days_plots.ecf

```diff
@@ -4,7 +4,7 @@
 #PBS -q %QUEUE%
 #PBS -A %PROJ%-%PROJENVIR%
 #PBS -l walltime=00:25:00
-#PBS -l place=vscatter:exclhost,select=1:ncpus=84:mem=100GB
+#PBS -l place=vscatter:exclhost,select=2:ncpus=88:mem=500GB
```

Comment:
@malloryprow The corresponding dev driver for this .ecf script doesn't include the `exclhost` part. Do we know which version of this line is correct and make the dev driver and .ecf script match?

Reply:
/lfs/h1/ops/prod/packages/evs.v1.0.13/ecf/scripts/plots/global_ens/jevs_global_ens_atmos_gefs_grid2obs_past90days_plots.ecf has `#PBS -l place=vscatter:exclhost,select=2:ncpus=88:mem=500GB` so we will got with that since that is what they decided to implement.